schema/enums/TaxIdentifierType.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/TaxIdentifierType.schema.json",
+  "title": "Enum - Tax Identifier Type",
+  "description": "Enumeration of tax ID types",
+  "type": "string",
+  "enum": ["SSN", "EIN", "ITIN", "PENDING", "OTHER"],
```

Comment:
This should probably be named as `UsTaxIdentifierType`, as the only options are applicable to United States.

We already have an issuing country code to help with tax identifier type (i.e. if the country code is CA, we assume it's a Canadian SIN), but what this seems to be doing is differentiating the many types of identifiers in use by the IRS

Reply:
Yeah, we wanted to add some structure to this so we could more reliably parse out common TAX IDs types and then do validation on the values. Do we want to have separate enums for different countries? Or do we want to use prefixes for now - e.g. US_SSN, US_EIN, CA_SIN, etc.?